hack/generator/pkg/astmodel/testdata/SetOptionalStringFromOptionalString-Direct.golden

```diff
@@ -14,11 +14,17 @@ func (person Person) ConvertFrom(source *vHub.Person) error {
 
 	// name
 	person.name = source.name
```

Comment:
This will still have source and destination sharing the name field - is that being fixed in a different change?

Reply:
Yes; I have a different PR (will put it out soon) that splits neither-optional from both-optional, and will fix the shared pointer issue there.